datadog/serializer.go

```diff
@@ -27,6 +27,8 @@ func (s *serializer) Write(b []byte) (int, error) {
 		return 0, io.ErrClosedPipe
 	}
 
+	b = ensureValidUTF8(b)
```

Comment:
Scratch that - we should be using `bytes.ToValidUTF8` here.

Reply:
this is amazing. didn't know this existed already.